packages/eui/src-docs/src/views/drag_and_drop/drag_and_drop_example.js

```diff
@@ -342,6 +345,56 @@ export const DragAndDropExample = {
       ),
       demo: <DragAndDropClone />,
     },
+    {
+      title: 'Portalled items',
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: dragAndDropPortalSource,
+        },
+      ],
+      text: (
+        <>
+          <p>
+            When using EUI drag components inside a stacking context, the fixed
+            drag positioning might be affected. To ensure dragging works as
+            expected inside e.g. <EuiCode>EuiFlyout</EuiCode> or{' '}
+            <EuiCode>EuiModal</EuiCode> use the prop{' '}
+            <EuiCode>usePortal</EuiCode> on <EuiCode>EuiDraggable</EuiCode>{' '}
+            components. This will render the currently dragged element inside a
+            portal on the body scope .
```

Comment:
```suggestion
            components. This will render the currently dragged element inside a
            portal appended to the document body (or wherever **EuiPortal** is configured to `insert` to by default).
```

Not sure if that last parenthetical there is super useful info, feel free to omit if so